src/openrct2/ride/Vehicle.cpp

```diff
@@ -6310,7 +6310,8 @@ static void AnimateSceneryDoor(const CoordsXYZD& doorLocation, const CoordsXYZ&
     {
         door->SetAnimationIsBackwards(isBackwards);
         door->SetAnimationFrame(1);
-        MapAnimationCreate(MAP_ANIMATION_TYPE_WALL_DOOR, doorLocation);
+        // NOTE: What is this doing here? This is probably not required.
```

Comment:
It starts the close door animation.

Reply:
I think I'll ditch this and just never delete the door animation or rather not mark the tile as not animated, it will also now only invalidate when the frame actually changes so there is probably no benefit now.